lintro/ai/providers/cursor.py

```
@@ -0,0 +1,253 @@
+            result = subprocess.run(
+                cmd,
+                input=combined_prompt,
+                capture_output=True,
+                text=True,
+                timeout=max(timeout, 300.0),
+                check=False,
+            )
+        except subprocess.TimeoutExpired as e:
+            raise AIProviderError(
+                f"Cursor CLI timed out after {timeout}s",
+            ) from e
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)